plate_simulation/driver.py

```diff
@@ -239,17 +240,38 @@ def make_model(self) -> FloatData:
             history=[dikes, overburden, erosion],
         )
 
-        geology = scenario.geologize()
-
-        if self.simulation_parameters.physical_property == "conductivity":
-            geology **= -1.0
+        geology, event_map = scenario.geologize()
 
         with fetch_active_workspace(self.params.geoh5, mode="r+"):
-            model: FloatData = self.mesh.add_data(  # type: ignore
-                {self.params.model.name: {"values": geology}}
+            value_map = {k: v[0] for k, v in event_map.items()}
+            physical_property_map = {k: v[1] for k, v in event_map.items()}
+
+            physical_property = self.simulation_parameters.physical_property
+            if physical_property == "conductivity":
+                physical_property_map = {
+                    k: 1 / v for k, v in physical_property_map.items()
+                }
+
+            model: ReferencedData = self.mesh.add_data(
+                {
+                    self.params.model.name: {
+                        "type": "referenced",
```

Comment:
I am not used to this code, so my questions are quite naives:

Before you were just adding 1 data and 1 object to the model, and now you're adding 2 of them. Why?

And they are sharing physical property map too.